lib/code-ribbon.js

```diff
@@ -49,18 +49,10 @@ module.exports = {
       'code-ribbon:cr-update': () => this.cr_update()
     }));
 
-    this.subscriptions.add(atom.commands.add('atom-workspace', {
-      'code-ribbon:add': () => this.add()
-    }));
-
-    this.subscriptions.add(atom.commands.add('atom-workspace', {
-      'code-ribbon:remove': () => this.remove()
-    }));
-
     // Register listener that watches config changes to pane_count_calc
     this.subscriptions.add(atom.config.onDidChange(
       "code-ribbon.pane_count_calc",
-      ({ newValue, oldValue }) => {
+      ({ newValue, oldValue }) => { // eslint-disable-line no-unused-vars
```

Comment:
I am guessing you can't get rid of that comment since ESlint will just remake it all the time

Reply:
No I put that comment there to prevent eslint from complaining about an error I intentionally made.

Reply:
very nice